features/step_definitions/web_ext_steps.rb

```diff
@@ -13,17 +13,17 @@
 
 Then /^I should see a link to "([^"]*)" with url (for|that starts with|that ends with) "([^"]*)"$/ do |name, attribute_selector, url|
   operator =
```

Comment:
Because the `operator` variable is no longer returning an "operator", we should rename it to something more accurate and descriptive. It will either return a regexp or a string, so something like `link_matcher` or `link_pattern`...?